src/tests/unit/tests/background/shortcuts-page-controller.test.ts

```diff
@@ -1,17 +1,35 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import { Mock, Times } from 'typemoq';
-
 import { ShortcutsPageController } from 'background/shortcuts-page-controller';
-import { BrowserAdapter } from '../../../../common/browser-adapters/browser-adapter';
+import { BrowserAdapter } from 'common/browser-adapters/browser-adapter';
+import { IMock, Mock } from 'typemoq';
+import { Tabs } from 'webextension-polyfill-ts';
 
 describe('ShortcutsPageController', () => {
-    it('opens the shortcuts tab', () => {
-        const browserAdapterMock = Mock.ofType<BrowserAdapter>();
+    let browserAdapterMock: IMock<BrowserAdapter>;
+    let testSubject: ShortcutsPageController;
+
+    beforeEach(() => {
+        browserAdapterMock = Mock.ofType<BrowserAdapter>();
+
+        testSubject = new ShortcutsPageController(browserAdapterMock.object);
+    });
+
+    it('opens the shortcuts tab', async () => {
+        browserAdapterMock
+            .setup(adapter => adapter.createActiveTab(ShortcutsPageController.configureCommandTabUrl))
+            .returns(() => Promise.resolve({} as Tabs.Tab));
+
+        await expect(testSubject.openShortcutsTab()).resolves.toBe(undefined);
```

Comment:
Should this be verifying that the call to `createActiveTab` actually happens?

Reply:
updated